src/main/java/io/icker/factions/command/AdminCommand.java

```diff
@@ -123,6 +138,9 @@ private int audit(CommandContext<ServerCommandSource> context) {
 
     public LiteralCommandNode<ServerCommandSource> getNode() {
         return CommandManager.literal("admin")
+                .requires(Requires.hasPerms("factions.admin.bypass",
+                    FactionsMod.CONFIG.REQUIRED_BYPASS_LEVEL))
```

Comment:
Using the GUI allows people to bypass the finer grained permissions requirements. I think the easiest solution is to just require the "factions.admin.gui" permission